packages/syft/src/syft/service/enclave/enclave_service.py

```diff
@@ -99,24 +99,22 @@ def get_all(
         roles=DATA_SCIENTIST_ROLE_LEVEL,
     )
     def request_enclave_for_code_execution(
-        self, context: AuthedServiceContext, service_func_name: str
+        self, context: AuthedServiceContext, service_func_id: UID
```

Comment:
To make this param more consistent with the rest of the code base, let's change this to `user_code_id: UID` everywhere instead.